internal/slicer/slicer_test.go

```diff
@@ -89,6 +89,7 @@ var slicerTests = []slicerTest{{
 						/other-dir/file: {symlink: ../dir/file}
 						/dir/text-file:  {text: data1}
 						/dir/foo/bar/:   {make: true, mode: 01777}
+						/var/lib/foo/**: {generate: manifest}
```

Comment:
[Nit]: Can you change the naming to match the existing convention used in the tests?